indico/modules/events/contributions/templates/forms/contribution_person_link_widget.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'forms/_person_link_widget_base.html' %}
+
+{% block sections %}
+    {% if field.allow_authors %}
+        {% set author_title = _('Authors') %}
+        {% set author_placeholder = _('There are no authors') %}
+        {{ render_section('author', author_title, author_placeholder) }}
+
+        {% set coauthor_title = _('Co-authors') %}
+        {% set coauthor_placeholder = _('There are no co-authors') if field.show_empty_coauthors else none %}
+        {{ render_section(id='coauthor', title=coauthor_title, placeholder=coauthor_placeholder) }}
+
+        {% set other_title = _('Others') %}
+        {{ render_section(title=other_title) }}
+    {% else %}
+        {% set placeholder = _('There are no speakers') %}
```

Comment:
why the extra assignment? you can just do `{ render_section(placeholder=_("There are no speakers")) }}`